### DISCUSSION TITLE
工作了几年再看代码写的很一般

### DISCUSSION THREAD

**Discussion by lilijreey** [chenshuo/muduo#587] [General]

**Comment by V0idk** [comment]
让人感觉写的一般也是一种功力啊.如果写的没人看得懂那才是完了

**Comment by shift4** [comment]
大道至简嘛，你搞个不一般的出来呀，小鬼。

**Comment by andylau004** [comment]
呵呵。。很一般？

**Comment by zinwalin** [comment]
Github Issues不是这样用的, 请尊重你的专业.

**Comment by NoevilMe** [comment]
360公司的网络库都显著声明参考了muduo ~~~

**Comment by anders0913** [comment]
呵呵

**Comment by lilijreey** [comment]
那我只能呵呵了

**Comment by TheWaveLab** [comment]
talk is cheap, show me the code

**Comment by mzygQAQ** [comment]
代码写的通俗易懂吧 这是好事

**Comment by FeronCompany** [comment]
我的项目星星能有陈硕一半我就心满意足了

**Comment by tomatowithpotato** [comment]
为啥这么说呢？来个分析呗，愿洗耳恭听

**Comment by lingr7** [comment]
写的一般，你正好可以改进啊，什么程序都写的完美了，还要码农干什么。show me your code.

**Comment by yinghaoyu** [comment]
有一说一，看过handy、sylar、muduo。handy用的c++11特性最多，sylar功能最全，muduo编码质量及风格最佳。
